client/src/model/apis/trainingsApi/trainingsApi.js

```diff
@@ -41,6 +41,14 @@ export const createTrainingsApi = ({ store, validator }) => {
   const deleteNewExercise = () => {
     store.newExercise = null;
   };
+  
+  const deleteExercise = (exId) => {
+    const data =  getData().newTraining;
+    store.newTraining = {
```

Comment:
you covered only flow with deleting from `newTraining` data.
you also need to cover flow with deleting from existing trainings (like it's done in `addExercise` function)

Reply:
if {
  // your code from the PR
} else {
  throw new Error('deleteExercise: deleting is not implemented yet!');
}